proof/crefine/RISCV64/ADT_C.thy

```diff
@@ -753,12 +777,16 @@ lemma cthread_state_rel_imp_eq:
   apply (cases y, simp_all add: ThreadState_defs)+
   done
 
-lemma ksPSpace_valid_objs_tcbBoundNotification_nonzero:
-  "\<exists>s. ksPSpace s = ah \<and> no_0_obj' s \<and> valid_objs' s
-     \<Longrightarrow> map_to_tcbs ah p = Some tcb \<Longrightarrow> tcbBoundNotification tcb \<noteq> Some 0"
+(* FIXME: choose better name for this lemma *)
+lemma ksPSpace_valid_objs_bound_objs_nonzero:
```

Comment:
I'm actually fine with the name for this lemma. From my side we could drop this FIXME.

Reply:
I think my concern was referring to the `tcbSchedPrev/Next` field with the term "bound". But I can remove this if it's still OK. Maybe `ksPSpace_valid_objs_refs_nonzero` would be better, if we do want to change it?

Reply:
You're right, `ksPSpace_valid_objs_refs_nonzero` is a better name.